src/lando/api/legacy/workers/base.py

```diff
@@ -135,12 +125,19 @@ def _start(self, max_loops: int | None = None, *args, **kwargs):
         # NOTE: The worker will exit when max_loops is reached, or when the stop
         # variable is changed to True.
         loops = 0
+        sleep_seconds = self.worker_instance.sleep_seconds
```

Comment:
This does not get updated on-the-fly, should probably move it to be in the loop.